docs/plat/ls1043a.rst

```diff
@@ -0,0 +1,88 @@
+Description
+===========
+
+The QorIQ® LS1043A processor is NXP's first quad-core, 64-bit Arm®-based processor for embedded networking.
+The LS1023A (two core version) and the LS1043A (four core version) deliver greater than 10 Gbps of performance
+in a flexible I/O package supporting fanless designs. This SoC is a purpose-built solution for small-form-factor
+networking and industrial applications with BOM optimizations for economic low layer PCB, lower cost power
+supply and single clock design. The new 0.9V versions of the LS1043A and LS1023A deliver addition power savings
+for applications such as Wireless LAN and to Power over Ethernet systems.
+
+LS1043ARDB Specification:
+-------------------------
+Memory subsystem:
+	* 2GByte DDR4 SDRAM (32bit bus)
+	* 128 Mbyte NOR flash single-chip memory
+	* 512 Mbyte NAND flash
+	* 16 Mbyte high-speed SPI flash
+	* SD connector to interface with the SD memory card
+
+Ethernet:
+	* XFI 10G port
+	* QSGMII with 4x 1G ports
+	* Two RGMII ports
+
+PCIe:
+	* PCIe2 (Lanes C) to mini-PCIe slot
+	* PCIe3 (Lanes D) to PCIe slot
+
+USB 3.0: two super speed USB 3.0 type A ports
+
+UART: supports two UARTs up to 115200 bps for console
+
+More information are listed in `ls1043`_.
+
+Boot Sequence
+=============
+
+
+Bootrom --> ATF BL1 --> ATF BL2 --> ATF BL1 --> ATF BL31
+--> BL32(Tee OS) --> ATF BL31 --> BL33(u-boot) --> Linux kernel
+
```

Comment:
The flow needs to be changes to  -

Bootrom -> ATF BL2-> ATF Bl31 -> BL32 ->Bl31 -> BL33

Additional BL1 step is not required on Layerscape platforms

Reply:
In this design, BL2 runs in EL1, so we must use BL1.  Maybe BL2 runs in EL3 is another option, it is not implemented in this patch.